src/main/java/it/gov/pagopa/pu/workflow/wf/sendnotification/wfsendnotification/SendNotificationProcessWF.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.pu.workflow.wf.sendnotification.wfsendnotification;
+
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+/**
+ * Workflow interface for Send Notification Workflow
```

Comment:
The javadoc of Workflow and Activity should explain what's the purpose of the component, not a generic description like this (a business logic description)